src/libaktualizr/primary/sotauptaneclient.cc

```diff
@@ -628,7 +628,7 @@ std::pair<bool, Uptane::Target> SotaUptaneClient::downloadImage(Uptane::Target t
     }
   }
   if (!success) {
-    LOG_ERROR << "Download unsuccessful after " << tries << " attempts.";
+    LOG_ERROR << "Download unsuccessful after " << tries - 1 << " attempts.";
```

Comment:
How about `for (; tries < max_tries; tries++)` ?

Reply:
Done.